apps/client/src/pages/builder/sidebars/left/sections/picture/section.tsx

```
@@ -22,20 +30,199 @@ import { PictureOptions } from "./options";
+      0,
+      size,
+      size
+    );
+
+    // Convert canvas to blob
+    canvas.toBlob(
+      async (blob) => {
+        if (!blob) {
+          <Toast variant="error">
+            Could not finalize the cropped image. Please try again.
+          </Toast>
```

ESLint

[error] String not marked for translation. Wrap it with <Trans>.

(lingui/no-unlocalized-strings)